tests/console/clamav.pm

```diff
@@ -40,6 +40,17 @@ sub run {
     $self->select_serial_terminal;
 
     zypper_call('in clamav vim');
+    zypper_call('info clamav');
+
+    # Check Clamav version
+    # Jira ID SLE-16780: upgrade Clamav SLE
+    my $current_ver = script_output("zypper info clamav | grep Version | cut -d':' -f2| cut -d'-' -f1");
+    record_info("Clamav_ver", "Current Clamav package version: $current_ver");
+
+    if ($current_ver < 0.101) {
+    record_soft_failure("SLE-16780 feature is not released");
```

Comment:
There are should be some "blank" before record_soft_failure for indentation.
Others are OK.

Reply:
sure, I miss to run the tidy, updated again. Thanks.